conf_files/huntsman.yaml

```
@@ -16,6 +16,7 @@ location:
 scheduler:
     type: dispatch
     fields_file: targets.yaml
+    check_file: True
```

Rename to make it clearer what it does. I guess this suggestion might have flow-on variables elsewhere in the code?
```suggestion
    reload_file_before_schedule: True
```

---

This comes down from the POCS side of things so we would need to make this change upstream.